docs/pages/components/select.md

```diff
@@ -25,7 +25,7 @@ For lists that require more than 12 options, the <a href="/patterns/combobox-inp
      <div class="fd-select">
          <div class="fd-select__control" role="button" tabindex="0" aria-controls="h0C6A325" aria-expanded="false" aria-haspopup="true">
              Select
-             <span class="fd-button fd-button--transparent sap-icon--slim-arrow-down fd-select__button"></span>
+             <button class="fd-button fd-button--transparent sap-icon--slim-arrow-down fd-select__button"></button>
```

Comment:
Do these need any aria-hidden attributes? aria-label? or does the placeholder give enough information?

Reply:
I think we should put a `tabIndex` of -1 on this. Otherwise we get two tab stops. The entire select should be a single tabstop as in this example: https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html 

That way it can be a button or a span, whichever is better for the visuals.